README.md

```diff
@@ -48,6 +48,43 @@ $ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/rel
 $ oc apply -f bundle/manifests
 ```
 
+### Create Project
+
+Create the project on jira-service-desk with the following custom resource:
+
```

Comment:
We can create, update and delete a Project using this operator. Kindly, update the documentation and also mention limitations like immutable fields cannot be updated, allowed project types etc.